GPU/Vulkan/GPU_Vulkan.cpp

```diff
@@ -205,11 +205,16 @@ void GPU_Vulkan::CheckGPUFeatures() {
 		if (!PSP_CoreParameter().compat.flags().DisableAccurateDepth || driverTooOld) {
 			features |= GPU_SUPPORTS_ACCURATE_DEPTH;
 		}
+		// These GPUs (up to some certain hardware version?) has a bug where draws where gl_Position.w == .z
+		// corrupt the depth buffer. This is easily worked around by simply scaling Z down a tiny bit when this case
+		// is detected. See: https://github.com/hrydgard/ppsspp/issues/11937
+		features |= GPU_NEEDS_Z_EQUAL_W_HACK;
```

Comment:
I wonder if these driver version checks ought to use the bugs_.Infest path.  Doesn't matter much, though.

-[Unknown]

Reply:
Probably. There are more things that should be on that path though, someday I'll clean it all up...